timed/reports/templates/mail/notify_supervisor_shorttime.txt

```diff
@@ -3,5 +3,5 @@ Time range: {{start}} - {{end}}
 Ratio: {{ratio}}
 
 {% for suspect, worktime in suspects %}
-{{suspect.get_full_name}} {{worktime.0}}/{{worktime.1}} (Ratio {{worktime.3|floatformat:2}} Balance {{worktime.2}})
+{{suspect.get_full_name}} {{worktime.0}}/{{worktime.1}} (Ratio {{worktime.3|floatformat:2}} Range Balance {{worktime.2}} Balance {{worktime.4}})
```

Comment:
Q: What is "range balance"?

Reply:
The balance of the given time range... have a better idea for a name?

Reply:
hours worked? If that is what it does

Reply:
no this is the balance between expected - reported in given time range.

Reply:
so, a delta then?

Reply:
Delta sounds good. Changed.